src/assets/modifierdata/ranger.yaml

```
@@ -182,6 +182,12 @@
           Expertise: [8, buff] # conversion of own 120 vitality
         conversion:
           Expertise: {Vitality: 7%}
+      wvwModifiers:
+        attributes:
+          Vitality: [120, buff]
+          Expertise: [8, buff] # conversion of own 120 vitality
```

```suggestion
          Expertise: [5, buff] # conversion of own 120 vitality
```
I assume the bonus expertise in WvW mode would be 0.04 * 120 = 5 instead of 8 here!

---

so the converion happens before the extra 120 vitaltiy and thats why i need to give the extra 5 or 8 epxterise?

---

It's a hacky bug fix on Arenanet's side, as far as we can tell. That 120 vitality isn't converted by anything else, but it apparently converts itself, according to ingame testing. So it looks like anet noticed it wasn't working how they wanted, but fixed it in a special way.